test/CMakeLists.txt

```diff
@@ -109,7 +109,7 @@ set(TEST_DEPENDS
 add_lit_testsuite(check-aie "Running the aie regression tests"
   ${CMAKE_CURRENT_BINARY_DIR}
   DEPENDS ${TEST_DEPENDS}
-  ARGS "-sv --timeout 600"
+  ARGS "-sv --timeout 600 -j4"
```

Comment:
This should really be cmake configurable..

Reply:
yea that's a flub - i keep adding and removing that

Reply:
https://github.com/llvm/llvm-project/pull/76205 - nice to be working on OSS tools...